docs/user/index.rst

```diff
@@ -4,34 +4,97 @@ Read the Docs: documentation simplified
 .. toctree::
    :maxdepth: 2
    :hidden:
-   :caption: 🚀 Tutorials
+   :caption: Getting Started
 
    /tutorial/index
+   /choosing-a-site
    /intro/getting-started-with-sphinx
    /intro/getting-started-with-mkdocs
    /intro/import-guide
-   /config-file/index
    /examples
 
 .. toctree::
    :maxdepth: 2
    :hidden:
-   :caption: 💡 Explanation
+   :caption: Project setup and configuration
 
-   /choosing-a-site
-   /integrations
-   /downloadable-documentation
+   /config-file/index
+   /config-file/v2
```

Comment:
I think this is great, having the related topics closer together makes it far easier to link all of these topics :+1: 

I could see us in the future adding another navigation layer so we could group minor topics all relating to big features, ie:

```yaml
- Project setup and configuration:
  - Configuration file:
    - Overview
    - Examples
    - Troubleshooting
    - Reference
```

Reply:
Agreed. There's definitely scope for additional pages 💯